aiogram/types/link_preview_options.py

```diff
@@ -22,6 +24,16 @@ class LinkPreviewOptions(TelegramObject):
     show_above_text: Optional[Union[bool, Default]] = Default("link_preview_show_above_text")
     """*Optional*. :code:`True`, if the link preview must be shown above the message text; otherwise, the link preview will be shown below the message text"""
 
+    @field_serializer(
+        "is_disabled",
+        "prefer_small_media",
+        "prefer_large_media",
+        "show_above_text",
+        when_used="json",
+    )
+    def serialize_fields(self, value):
+        return value.__str__()
```

Comment:
You should never call the magic methods directly, just use `str(...)` instead.

Reply:
fixed